### PR TITLE
Fix lack of curl for lore job

### PR DESCRIPTION
**Pull request by rattboi**
Signed-off-by: Bradon Kanyid <rattboi@gmail.com>

### CODE REVIEW

**Review by omgjlk — APPROVED** [review]

File: roles/zuul/files/jobs/lore.yaml [hunk 1]
```diff
@@ -6,8 +6,9 @@
       - zuul-git-prep
       - shell: |
           #!/bin/bash -ex
+          sudo apt-get install -y curl
           curl -sL https://deb.nodesource.com/setup_7.x | sudo -E bash -
-          sudo apt-get install -qq nodejs
+          sudo apt-get install -y nodejs
```
Comment (mlangbehn) [inline roles/zuul/files/jobs/lore.yaml:11]: Are we sure we want this much output?

Comment (eggshell) [reply roles/zuul/files/jobs/lore.yaml:11]: At this point, seeing it might not be terrible. We can always turn it back off once this has settled.

**Comment by rattboi** [comment]
gate_recheck

**Comment by rattboi** [comment]
gate-recheck

**Comment by anne-bonny** [comment]
Starting gate_github jobs.

File: roles/zuul/files/jobs/lore.yaml (hunk 1 above)
Comment (mlangbehn) [reply roles/zuul/files/jobs/lore.yaml:11]: Oh wait! This will still fail because we aren't installing npm.

**Pull request merged**